pokemongo_bot/cell_workers/collect_level_up_reward.py

```diff
@@ -70,3 +70,5 @@ def _collect_level_reward(self):
                     'items': data
                 }
             )
+            # if successfully collect reward, update previous level
+            self.previous_level = self.current_level
```

Comment:
We already have this line in `work()`. Do we need this line again in `_collect_level_reward()` which we call in `work()`?